sentry/src/main/java/io/sentry/Scope.java

```diff
@@ -1076,16 +1093,17 @@ public void setPropagationContext(final @NotNull PropagationContext propagationC
   @ApiStatus.Internal
   @Override
   public @NotNull PropagationContext getPropagationContext() {
-    return propagationContext;
+    return propagationContext.getValue();
   }
 
   @ApiStatus.Internal
   @Override
   public @NotNull PropagationContext withPropagationContext(
```

Comment:
What's the use case of returning a clone of `PropagationContext` here? As far as I can see we at least internally never seem to use the return type.